person_capture/gui_app.py

```
@@ -4460,11 +4476,12 @@ def _atomic_jpeg_write(img: np.ndarray, out_path: str, q: int) -> tuple[bool, st
                         fh.write(buf.tobytes())
-                        fh.flush()
-                        try:
-                            os.fsync(fh.fileno())
-                        except Exception:
-                            pass
+                        if bool(getattr(self.cfg, "save_fsync", False)):
+                            fh.flush()
+                            try:
+                                os.fsync(fh.fileno())
+                            except Exception:
+                                pass
```

Ruff (0.15.12)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)